app/models/event.rb

```diff
@@ -0,0 +1,41 @@
+class Event
+  DESCRIPTION_CUT = 140
+
+  def initialize(raw_event_details)
+    @_name = raw_event_details["title"]
+    @_date_time = raw_event_details["start_time"]
+    @_description = raw_event_details["description"]
+    @_venue_name = raw_event_details["venue_name"]
+    @_venue_url = raw_event_details["venue_url"]
+    @_venue_address = raw_event_details["venue_address"]
+  end
+
+  def name
```

Comment:
Thank you @case-eee for the comment. I liked Josh's suggestion from one of our classes to mark up instance variables this way, which will not be directly exposed outside of the `Event` object and that is why I decided to implement them this way.